### DISCUSSION TITLE
40mm Acid Grenade icon is horizontal whilst every other 40mm grenade is diagonal

### DISCUSSION THREAD

**Discussion by Wheatley1001** [FakeFishGames/Barotrauma#11241] [Ideas and suggestions]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Super minor inconsistency here, wanted to point it out regardless.
![image](https://user-images.githubusercontent.com/105495756/218345764-dfd717bc-88df-4a73-a073-9895c300b26a.png)

### Bug prevalence

Happens every time I play

### Version

Faction/endgame test branch

### Which operating system did you encounter this bug on?

Windows

**Comment by HawkinsThe71st** [comment]
Would be nice if Acid Grenades and 40mm Acid Grenades had inventory icons with the same orientation as the rest of the grenades/throwables
![image](https://user-images.githubusercontent.com/115734437/219019862-5760f789-f68a-4740-8658-84bd959154be.png)
